### DISCUSSION TITLE
Please, add to output time taken to render image. In the log also, if possible.

### DISCUSSION THREAD

**Discussion by ptits** [MoonRide303/Fooocus-MRE#23] [General]
![shot_230823_214522](https://github.com/MoonRide303/Fooocus-MRE/assets/13587736/ca1ae7d5-680e-44a9-af45-13d132823d3a)
            start_time = time.time()

            imgs = pipeline.process(p_txt, n_txt, steps, switch, width, height, seed, sampler_name, scheduler,
                cfg, base_clip_skip, refiner_clip_skip, input_image_path, start_step, denoise, callback=callback)
            end_time = time.time()
            elapsed_time = end_time - start_time
            print(f"\033[91mTime taken: {elapsed_time} seconds\033[0m")

**Comment by 3Diva** [comment]
This would be great! I find that it's super helpful in both ComfyUI and A1111 that you can see the image generation time for each image. This makes it MASSIVELY helpful when testing out different settings to find the ones that produce the quickest results. Having the ability to see image generation times with Fooocus would be very helpful!

**Reply by 3Diva** [reply]
What is this magic I see!? "Prompt executed in _______"! Thank you so much for the awesome QOL update, MoonRide303!
![Screenshot 2023-09-02 152928](https://github.com/MoonRide303/Fooocus-MRE/assets/51475772/10e2ea62-343e-4032-b232-ea72f314ec8a)
